.github/workflows/service-py-deb-workflow.yml

```diff
@@ -330,64 +330,39 @@ jobs:
     if: ${{ inputs.release == 'false' }}
```

Comment:
Debian package does not contain the correct version, see the step where bumpversion is called and try something like this if you wanna keep calculating the version 
```
	bump2version --no-tag --allow-dirty --new-version $(PACKAGE_VERSION)
```

or just `bump2version build --no-tag --allow-dirty --commit-args no-verify` to bump the build id only